rcl/CMakeLists.txt

```diff
@@ -121,6 +128,8 @@ ament_export_dependencies(rcutils)
 ament_export_dependencies(${RCL_LOGGING_IMPL})
 ament_export_dependencies(rosidl_runtime_c)
 ament_export_dependencies(tracetools)
+ament_export_dependencies(rosidl_typesupport_c)
```

Comment:
I guess we don't need to export typesupport_c since it is not exposed in any public headers.